server/controllers/userController.js

```diff
@@ -168,9 +168,9 @@ class UserController {
         // Push the new game profile's ObjectId to the user's gameProfiles array
         userProfile.gameProfiles.push(newGameProfile._id);
       }
-  
       // Save the UserGameProfile document
       await userProfile.save();
+      res.send({userId, gameUserId, name, username, homeState, country, dob, game, role, status, team},{"message": "Game Profile Changed successfully" })
```

Comment:
it is not needed and you won't get this data everytime. you will only get this data when a new profile is created.